docs/src/main/sphinx/installation/cli.rst

```diff
@@ -80,6 +80,35 @@ The detailed behavior is as follows:
   authentication system configuration.
 * Expired tokens force you to log in again.
 
+.. _cli-certificate-auth:
+
+Certificate authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the following command line arguments to allow you to connect the CLI to a
+cluster that enables client :doc:`certificate authentication
+</security/certificate>`.
+
+.. list-table:: Certificate authentication options
+   :widths: 35 65
+   :header-rows: 1
+
+   * - Option
+     - Description
+   * - ``--keystore-path=<path>``
+     - Keystore path to a :doc:`PEM </security/inspect-pem>` or :doc:`JKS
+       </security/inspect-jks>` file, which must include a certificate and
+       private key for the CLI.
+   * - ``--keystore-password=<password>``
+     - Only required if the key has a password.
```

Comment:
Is that the password for the key or the keystore file? Or is that the same? In either case something like

Password to access the keystore file.

or

Password to access the certificate in the keystore file.

Reply:
keystore pw